_content/booklets/refreshing-translation-of-the-suttanipata_sujato.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Anthology of Discourses: a Refreshing Translation of the Suttanipāta"
+authors:
+  - sujato
+external_url: "https://suttacentral.net/edition/snp/en/sujato?lang=en"
+drive_links:
+  - "https://drive.google.com/file/d/1-HBJF5CAUcgr1bDCT0wOLT6dbaZ6Cjdp/view?usp=sharing"
+course: snp
+tags:
+  - pali-canon
+year: 2023
+month: jan
+pages: 399
+---
+
+> The obsession with seeing portions of the Suttanipāta as early
+is a holdover of the mid-20th century enthusiasm for discovering
+a “Buddha before Buddhism”, seeking a “truly authentic” teaching
+before it was institutionalized as rigid doctrine. Somehow, this
+search always ends up conflated with the racially-charged effort to
+divest Buddhism of its “cultural” (read “Asian”) elements. The Suttanipāta does not need rescue, and a romanticizing and reductive
+lens adds nothing and distorts much.
+
+This collection of teachings from the Suttanipāta contains 74 short texts in verse or mixed prose and verse, arranged in five chapters. It contains some of the most well-known texts in popular Buddhism, such as the Ratana, Maṅgala, and Mettā Suttas, and texts such as the “Rhinoceros Horn” and the “Way to the Beyond” are regarded as among the earliest of the Buddhist scriptures.
```

Comment:
Wrong. This isn't a collection of teachings from the Snp... it is the entire Snp.